src/app/admin/admin-reports/filtered-collections/filtered-collections.component.ts

```diff
@@ -0,0 +1,90 @@
+import { Component, ViewChild } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { NgbAccordion } from '@ng-bootstrap/ng-bootstrap';
+import { Observable } from 'rxjs';
+import { RestRequestMethod } from 'src/app/core/data/rest-request-method';
+import { DspaceRestService } from 'src/app/core/dspace-rest/dspace-rest.service';
+import { RawRestResponse } from 'src/app/core/dspace-rest/raw-rest-response.model';
+import { environment } from 'src/environments/environment';
+import { FiltersComponent } from '../filters-section/filters-section.component';
+import { FilteredCollections } from './filtered-collections.model';
+
+@Component({
+  selector: 'ds-report-filtered-collections',
+  templateUrl: './filtered-collections.component.html',
+  styleUrls: ['./filtered-collections.component.scss']
+})
+export class FilteredCollectionsComponent {
```

Comment:
It'd be nice to add a brief comment at the top of this class to describe the purpose of this class. It helps other developers to understand where the component is being used & what it does.

Reply:
Done.